sys/posix/sockets/posix_sockets.c

```diff
@@ -68,6 +68,7 @@ typedef struct {
     int type;
     int protocol;
     bool bound;
+    uint32_t recv_timeout;
```

Comment:
ok fine, I also guarded any reference to this variable. So if ```POSIX_SETSOCKOPTS``` is not defined,
it does not change the current behaviour.